nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/WriteAheadFlowFileRepository.java

```diff
@@ -129,17 +138,22 @@ public WriteAheadFlowFileRepository() {
         checkpointDelayMillis = 0l;
         numPartitions = 0;
         checkpointExecutor = null;
-        flowFileRepositoryPath = null;
+        walImplementation = null;
```

Comment:
Should this be DEFAULT_WAL_IMPLEMENTATION?

Reply:
It doesn't really matter what it's set to - that constructor will only be used for service loading and no method in the class will ever be called.